pkg/util/image_utils.go

```diff
@@ -67,3 +68,9 @@ func copyToFile(outfile string, r io.Reader) error {
 
 	return nil
 }
+
+// checks to see if an image string contains a tag.
+func HasTag(image string) bool {
+	tagRegex := regexp.MustCompile(".*:[^/]*$")
```

Comment:
can you add some unit tests for the regex?

Reply:
done